core/trino-main/src/main/java/io/trino/execution/scheduler/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -366,7 +373,8 @@ public synchronized void start()
                     smallStageSourceSizeMultiplier,
                     smallSizePartitionSizeEstimate,
                     stageEstimationForEagerParentEnabled,
-                    smallStageRequireNoMorePartitions);
+                    smallStageRequireNoMorePartitions,
+                    markDistinctChainEstimationEnabled);
```

Comment:
below, `markDistinctChainEstimationEnabled` comes after `stageEstimationForEagerParentEnabled` (eg in Scheduler fields).
use same order here?

Reply:
Cleaned this up (moved `stageEstimationForEagerParentEnabled` to proper place)